File: Android/phoneNode/src/testStandalone/java/com/chebysight/chebyagent/android/StandaloneBuildIdentityTest.java
package com.chebysight.chebyagent.android;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;

import org.junit.Test;

public final class StandaloneBuildIdentityTest {
    @Test
    public void standaloneBuildUsesAnIsolatedApplicationIdentity() {
        assertEquals(
                "com.chebysight.chebyagent.phonenode.standalone",
                BuildConfig.APPLICATION_ID
        );
        assertTrue(BuildConfig.STANDALONE_MODE);
        assertEquals("ChebyNode Standalone", BuildConfig.AGENT_NODE_PRODUCT_NAME);
        assertEquals(
                "http://127.0.0.1:3448",
                BuildConfig.PHONEBRIDGE_DEFAULT_BASE_URL
        );
    }
}

File: Android/app/src/testStandalone/java/com/cheby/codex/mobile/StandaloneBuildIdentityTest.kt
package com.cheby.codex.mobile

import org.junit.Assert.assertEquals
import org.junit.Assert.assertFalse
import org.junit.Assert.assertTrue
import org.junit.Test

class StandaloneBuildIdentityTest {
    @Test
    fun standaloneBuildCannotReuseDistributedIdentityOrRelayFallbacks() {
        assertEquals("com.cheby.codex.mobile.standalone", BuildConfig.APPLICATION_ID)
        assertTrue(BuildConfig.STANDALONE_MODE)
        assertFalse(BuildConfig.DEMO_GATEWAY_ENABLED)
        assertFalse(BuildConfig.LEGACY_RELAY_MIGRATION_ENABLED)
        assertEquals("", BuildConfig.LEGACY_RELAY_ORIGIN)
    }
}
